src/page-viewer/styles.css

```diff
@@ -0,0 +1,6 @@
+.root {
+    float: right;
+    padding: 4px 4px 4px 12px;
```

Comment:
That sounds like the best way to do it, I think common practice is to setup your components with some reasonable defaults and have everything else passed in by the caller.